cmds/dump.c

```diff
@@ -879,6 +879,25 @@ static void dump_chrome_task_rstack(struct uftrace_dump_ops *ops, struct uftrace
 	size_t len = sizeof(name_buf) - 1;
 	size_t i;
 	struct uftrace_dbg_loc *loc = NULL;
+	uint64_t timestamp;
+	static uint64_t utc_offset_ns = 0;
+	static bool offset_initialized = false;
+
+	/* Initialize the UTC offset once if the option is enabled */
+	if (chrome->opts && chrome->opts->with_utc_offset && !offset_initialized) {
```

Comment:
I don't think chrome->opts can be NULL.  Also I think we can just use `utc_offset_ns == 0` if it's initialized or not.